src/python/big5_personality/big5.py

```diff
@@ -44,13 +46,28 @@ def clean_up(message_bodies):
         clean_message_body = re.sub(r'On\s(.[^\sw]*\s)*wrote', '', clean_message_body, flags=re.MULTILINE)
         clean_message_body = re.sub(r'[\n+]Sent from', '', clean_message_body, flags=re.MULTILINE)
         clean_message_body = re.sub(r'https?:\/\/\S*', '', clean_message_body, flags=re.MULTILINE)
-    clean_message_body = re.sub(r'[\w\.-]+ @ [\w\.-]+', '', clean_message_body, flags=re.MULTILINE)
+        clean_message_body = re.sub(r'[\w\.-]+ @ [\w\.-]+', '', clean_message_body, flags=re.MULTILINE)
         # clean_message_body = clean_message_body.encode('utf-8').strip()
 
+        split_message = clean_message_body.split()
+        words_number = words_number + len(split_message)
+        if words_number > words_limit:
+            split_message = cutoff(split_message, words_number-words_limit)
```

Comment:
questo codice è poco 'pythonic'. si puo' fare tutto in una sola istruzione con un solo metodo usando l'array slicing: https://stackoverflow.com/questions/509211/understanding-pythons-slice-notation